File: problems/math-linter/scaffold.java
import java.util.*;
import java.util.regex.Pattern;

public class solution {
    private static boolean validateExpression(String[] tokens) {
        // Your solution goes here
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] tokens = scanner.nextLine().split(" ");
        boolean isValid = validateExpression(tokens);
        System.out.println(isValid ? "true" : "false");
    }
}

File: problems/11-math-linter/scaffold.cpp
#include <iostream>
#include <string>
#include <vector>
#include <regex>
#include <algorithm>

using namespace std;

bool validateExpression(vector<string> tokens)
{
    // Your solution goes here
}

int main()
{
    string input;
    getline(cin, input);
    vector<string> tokens;
    stringstream ss(input);
    string temp;
    while (getline(ss, temp, ' '))
    {
        tokens.push_back(temp);
    }
    bool isValid = validateExpression(tokens);
    cout << (isValid ? "true" : "false") << endl;
    return 0;
}